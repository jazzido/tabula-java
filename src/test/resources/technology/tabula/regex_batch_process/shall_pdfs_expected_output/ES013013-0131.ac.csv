GASTON CO ENVIRONMENTAL HEALTH,,ALVIN JOHNSON,
991 WEST HUDSON BLVD,,,
"",,208 CREST LN.,
"GASTONIA, NC 28052 Courier # 06-34-01",,"BELMONT, NC 28012",
EIN: 566000300EH,,,
StarLiMS ID: ES013013-0131001 Date Collected:,01/28/13,Time Collected:,12:59 PM
Date Received:,01/30/13,Collected By:,Ronald Whisnant
Sample Type: Sampling Point:,Sample head,Well Permit #:,10101
Analyte,,Result,Allowable Limit,Unit Qualifier(s)
Nitrate,,3.70,10.00,mg/L
Nitrite,,< 0.10,1.00,mg/L
Report Date:,02/12/2013,,Reported By:,Arnold Hall
