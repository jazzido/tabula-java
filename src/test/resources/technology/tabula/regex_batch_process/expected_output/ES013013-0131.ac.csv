Nitrate,3.70,10.00,mg/L
Nitrite,< 0.10,1.00,mg/L
